Flipdish.Recruiting.iOS/README.md

```diff
@@ -0,0 +1,54 @@
+We are happy you're interested in an engineering position at Flipdish!
+
+### About This Challenge
+We want to see how you construct software, but we also know that you're busy. So, rather than ask you to build something from scratch, we want you to take whatever time is reasonable and refactor some of the code in this application. Just make it better, in literally any way that you see fit.
+
+It doesn't matter how long you spend on improving it! You could spend 5 minutes, or 5 hours -- the amount of time is totally up to you. It'll really help us understand your perspectives on software when we see how you improve existing code that you haven't encountered before. Feel free to refactor, change, improve, rewrite or optimize any part of the application at all.
+
+### About The app
+
+The app is a very simple List / Detail client for the [The Dog API](https://docs.thedogapi.com/). It works (for the most part), but the code is truly, madly and deeply squalid. As you'll quickly notice, the app is far from perfect in many different ways - Let's make it better!
+
+##### Functionality
+
+- The basic premise is to fetch a list of Dogs from the API, and render them in a list.
+- Each list item can be opened for a detail view and from there the user can favourite their favourite Perros 🦮🐕‍🦺🐕🐩.
+- There's a section at the top of the list view with quick access to the user's favourited Dogs.
+
+See attached screenshots for how the app <em>should</em> look.
```

Comment:
Does this `<em>` work in GitHub markdown? That's where people will read this file.

Reply:
It does surprisingly! For italicising, if you open this page you should see it working https://github.com/flipdishbytes/recruiting/tree/add-ios-recruiting-project/Flipdish.Recruiting.iOS